components/ntp_background_images/browser/view_counter_service.cc

```diff
@@ -76,10 +76,11 @@ ViewCounterService::ViewCounterService(NTPBackgroundImagesService* service,
   branded_new_tab_count_state_ =
       std::make_unique<WeeklyStorage>(local_state, kSponsoredNewTabsCreated);
 
-  if (auto* data = GetCurrentBrandedWallpaperData())
-    model_.set_total_branded_image_count(data->backgrounds.size());
-  if (auto* data = GetCurrentWallpaperData())
-    model_.set_total_image_count(data->backgrounds.size());
+  ResetModel();
+  // if (auto* data = GetCurrentBrandedWallpaperData())
```

Comment:
did you mean to keep this commented-out code in?

Reply:
It should be deleted :)
Deleted by https://github.com/brave/brave-core/pull/11628